CMakeLists.txt

```diff
@@ -67,6 +75,9 @@ add_library(puzzles_lib OBJECT
 
 # Executable
 add_executable(puzzles src/main.cpp $<TARGET_OBJECTS:puzzles_lib>)
+add_custom_target(run "./puzzles"
```

Comment:
Is it necessary to break this in 2 lines? feels a bit weird

Reply:
Great point, fixed! I actually don't remember why I added the line break - I usually do it when there are multiple properties, but that's not the case here. Probably some leftover from wip code hehe